foodpicker.py

```diff
@@ -0,0 +1,52 @@
+# Updated the file.
+# Created a dictionary of 10 foods and their ingredients.
+
+food_ingredients = {
+    "Pizza": ["dough", "tomato sauce", "cheese", "pepperoni"],
+    "Burger": ["bun", "beef patty", "cheese", "lettuce", "tomato", "onion", "ketchup", "mayonnaise"],
+    "Fried Chicken": ["chicken", "flour", "breadcrumbs", "spices", "oil"],
+    "Macaroni and Cheese": ["macaroni pasta", "cheese", "butter", "milk", "flour"],
+    "Hot Dog": ["hot dog bun", "sausage", "ketchup", "mustard", "relish", "onion"],
+    "Pancakes": ["flour", "milk", "eggs", "baking powder", "butter", "syrup"],
+    "Grilled Cheese Sandwich": ["bread", "cheese", "butter"],
+    "Chicken Wings": ["chicken wings", "hot sauce", "butter", "celery", "blue cheese dressing"],
+    "French Fries": ["potatoes", "oil", "salt"],
+    "Salad": ["romaine lettuce", "Caesar dressing", "parmesan cheese", "croutons"],
+}
+
+# Created a function that makes a list of the user entered ingredients and limit it to ALL lower case format.
+
+def find_food_item(ingredient):
+    matches = []
+    for food, ingredients in food_ingredients.items():
+        if ingredient.lower() in [i.lower() for i in ingredients]:
+            matches.append(food)
+    return matches
+
+# Created a function that will take the user submitted list of ingredients and make sure each item in the list matches for the intended food item.
+
+def can_make_food(food_item, ingredients):
+    required_ingredients = food_ingredients.get(food_item, [])
```

Comment:
why default to an empty list? `food_item` is passed in from a function that is using the food_items from the dict at the top. Don't need a default.